frontend/src/concepts/pipelines/content/tables/pipelineRun/useMetricColumns.ts

```diff
@@ -52,10 +54,15 @@ export const useMetricColumns = (
   runs: RunWithMetrics[];
   metricsColumnNames: string[];
   runArtifactsLoaded: boolean;
+  contextError: Error | undefined;
   runArtifactsError: Error | undefined;
   metricsNames: Set<string>;
 } => {
-  const [runArtifacts, runArtifactsLoaded, runArtifactsError] = useGetArtifactsByRuns(runs);
+  const [context, , contextError] = useMlmdContextsByType(MlmdContextTypes.RUN);
```

Comment:
Nit: for naming convention, we'd better call it `contexts` and `contextsError` here, since it returns the array.